src/ansys/pytwin/evaluate/twin_model.py

```diff
@@ -975,6 +1333,114 @@ def save_state(self):
             msg += f"\n{str(e)}."
             self._raise_error(msg)
 
+    def snapshot_generation(self, rom_name: str, on_disk: bool = True, named_selection: str = None):
+        """
+        Generate a field snapshot based on current states of the Twin, either in memory or on disk, for the full field
+        or a specific part. It returns the field data as an array if in memory, or the path of the snapshot written on
+        disk.
+
+        Parameters
+        ----------
+        rom_name: str
+            TBROM name part of the Twin for which a snapshot has to be generated
```

Comment:
```suggestion
            TBROM name part of the twin to generate the snapshot on.
```
I don't understand why "part" is included in the description--so my suggestion here may be incorect.